src/mailadm/cmdline.py

```diff
@@ -179,80 +177,32 @@ def gen_qr(ctx, tokenname):
               default="https://example.org/new_email", show_default="https://example.org/new_email")
 @click.option("--mail-domain", type=str, prompt="mail domain for which we create new users",
               default="example.org", show_default="example.org")
-@click.option("--vmail-user", type=str, default="vmail",
-              help="dovecot virtual mail delivery user")
-@click.option("--path-virtual-mailboxes", default=None, type=str,
-              help="postfix virtual users map, generated by mailadm")
+@click.option("--mailcow-endpoint", type=str, default=None,
+              help="the API endpoint of the mailcow instance")
+@click.option("--mailcow-token", type=str, default=None,
+              help="you can get an API token in the mailcow web interface")
```

Comment:
i think you can teach click to also read from "MAILCOW_ENDPOINT" and "MAILCOW_TOKEN" environment variables if nothing is specified on command line.

Reply:
cooool, that's super useful. I wonder whether it makes sense beyond the init subcommand... on first glance it doesn't